.github/workflows/style-guide.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Verify style guide compliance on pull request
+on: [pull_request]
+
+jobs:
+  vale:
+    name: Linting with Vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Asciidoctor
+        run: sudo apt-get install -y asciidoctor
+      - uses: errata-ai/vale-action@reviewdog
+        with:
+          filter_mode: diff_context
+          vale_flags: "--no-exit --minAlertLevel=error"
+          reporter: github-pr-review
+          fail_on_error: true
+        env:
+          # Required, set by GitHub actions automatically:
+          # https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret
+          GITHUB_TOKEN: ${{ secrets.RHDH_BOT_TOKEN }}
+          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.RHDH_BOT_TOKEN }}
```

Comment:
do you need these envs? https://github.com/errata-ai/vale-action?tab=readme-ov-file#token-default-secretsgithub_token

Reply:
Updated